tests/subsys/bluetooth/mesh/light_ctrl/testcase.yaml

```diff
@@ -0,0 +1,6 @@
+tests:
+  bluetooth.mesh.light_ctrl_subsys:
+    platform_allow: native_posix
+    tags: bluetooth mesh models light_ctrl
```

Comment:
Tags are used to group tests together on root level. When invoking twister, you can filter tests by tag by calling `twister -t tag-name`.

In sdk-nrf, we have a special tag `ci_build`, as well as top level system tags, such as `bluetooth`. 
I realize that this is just following the same practice as the existing mesh tests, but to avoid adding to the issue, I suggest we only put `bluetooth` and `ci_build` tags on this, and I'll make a PR to remove the tags from the other tests.